### PR TITLE
Fixes all game/battlefield CSS

### PR DESCRIPTION
**Pull request by acpennington**

### CODE REVIEW

File: web-app/src/components/YugiohCard/YugiohCard.js [hunk 1]
```diff
@@ -138,8 +138,9 @@ function YugiohCard({ height, notFull, player, row, zone, cardName, modal, isHer
       };
    }, []);
 
-   const margin = !notFull && (height - height / CARD_RATIO) / 2 + 2;
+   const margin = !notFull && (height - height / CARD_RATIO) / 2 + 2; // the +2 is to leave a little space between cards
```
Comment (scheibo) [inline web-app/src/components/YugiohCard/YugiohCard.js]: I think you're not flooring here when you want to be?

```suggestion
   const width = Math.floor(height / CARD_RATIO),
   const margin = !notFull && (height - width) / 2 + 2; // the +2 is to leave a little space between cards
```

And then replace `width: Math.floor(height / CARD_RATIO),` with just `width`.

File: web-app/src/views/Game/Battlefield/Battlefield.js [hunk 2]
```diff
@@ -1,100 +1,59 @@
...
 
 class Battlefield extends Component {
    render() {
-      const { classes, handCounts, players, size, player } = this.props;
+      const { classes, handCounts, players, rowHeight, player } = this.props;
       const { solo } = player;
       const HERO = players.hero;
       const VILLAIN = players.villain;
-      const cardHeight = Math.floor(size - 2);
+      const cardHeight = Math.floor(rowHeight - 2); // this is the most straightforward way to leave some space between cards
```
Comment (scheibo) [inline web-app/src/views/Game/Battlefield/Battlefield.js:23]: Isn't the most straightforward way to just add 2px margins between the `cardRow` in CSS?

File: web-app/src/views/Game/Battlefield/Hand.js [hunk 3]
```diff
@@ -8,18 +8,19 @@ import FriendlyScroll from "components/FriendlyScroll/FriendlyScroll.js";
...
 
-function Hand({ player, handCount, size, isHero, revealed, phase }) {
+function Hand({ player, handCount, rowHeight, isHero, revealed, phase }) {
    const classes = useStyles();
    const dispatch = useDispatch();
    const socket = useContext(WebSocketContext);
+   const handWidth = rowHeight * 5;
 
-   const handSize = handCount > 7 ? (isHero ? size * 0.95 : size * 0.9) : size;
+   const handRowHeight = handCount > 7 ? (isHero ? rowHeight * 0.95 : rowHeight * 0.9) : rowHeight;
```
Comment (scheibo) [inline web-app/src/views/Game/Battlefield/Hand.js]: Let's add a comment to explain that this is intended to account for friendly scroll's scroll bar height?

File: web-app/src/views/Game/Battlefield/Battlefield.js (hunk 2 above)
Comment (acpennington) [reply web-app/src/views/Game/Battlefield/Battlefield.js:23]: If you add 2px margins and leave the cards the same size, then the bottom 2px of each card will be cut off.

File: web-app/src/components/YugiohCard/YugiohCard.js (hunk 1 above)
Comment (acpennington) [reply web-app/src/components/YugiohCard/YugiohCard.js]: This does make more sense. I'll do it this way.

File: web-app/src/views/Game/Battlefield/Battlefield.js (hunk 2 above)
Comment (scheibo) [reply web-app/src/views/Game/Battlefield/Battlefield.js:23]: this feels wrong to me but i'll have to play around with it myself once its committed to understand.

**Review by scheibo — APPROVED** [review]

**Pull request merged**